stable/yugaware/templates/statefulset.yaml

```diff
@@ -98,6 +96,11 @@ spec:
         - name: postgres
           image: {{ include "full_image" (dict "containerName" "postgres" "root" .) }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.securityContext.enabled }}
+          securityContext:
+            runAsUser: 999
```

Comment:
I understand the Postgres user in the container image is with the UID `999`. This might break the upgrade path if someone is already running as a non-root with a different UID. Instead of hard-coding it, it's better to externalize in the values.yaml.

Reply:
It won't happen unless they have modified the platform charts according to their needs. Because as per old changes, the platform charts do not change the user for the Postgres container. However, as per new changes, it will change the user for the Postgres container if we want. Otherwise, it'll use 999 as default.